consensus/src/pending_votes.rs

```diff
@@ -29,7 +29,7 @@ use std::{
 pub enum VoteReceptionResult {
     /// The vote has been added but QC has not been formed yet. Return the amount of voting power
     /// QC currently has.
-    VoteAdded(u64),
+    VoteAdded(u128),
```

Comment:
Why did this change?

Reply:
sum of u64 can overflow u64, it's now aligned with the total supply (u128)